### ISSUE TITLE
Fix 409 Git Repository is empty.

### ISSUE THREAD

**Issue by jefftriplett** [djangopackages/djangopackages#728]
Occasionally a package gets moved or deleted we get a 409 error from github in our logs when the`manage.py package_updater` command is run. 

```
For dwangoapp, <class 'github3.models.GitHubError'>: 409 Git Repository is empty.
```

We should archive these and maybe log a reason so that we can stop checking GH every day for updates. They should also be removed from Grids and possible Search too if we can't locate where they went.

**Comment by kishan3** [comment]
Hi, @jefftriplett I would like to understand which part of the project I should check for this.

**Comment by jefftriplett** [comment]
@kishan3 thanks for the following up. We can use Python logging (should already be imported) and use `logger.error("some meaningful message here")` to log a reasonable error message. 

We can probably add an `active  = BooleanField(default to true)` and toggle it to False when we run into this 409.  

I'll think about this some more and I might do a quick follow up PR because that active bit is going to touch a lot of places.

**Comment by kishan3** [comment]
Thanks, @jefftriplett I created a basic PR You can advise if I need to do something more for this.

**Comment by jefftriplett** [comment]
Fixed in #733
